app/views/static/manuscript_descriptions.html.erb

```diff
@@ -20,32 +20,51 @@
+        </p>
+
+        <p class='static-page__text static-page__text--sinai'>
+          Additionally, and where feasible, we supply uniform titles for textual works. Providing an authoritative form for titles allows a user to collocate all versions for a given work, especially where the title or spelling might differ between versions. In some cases we may also supply an alternative title for a work, such as a supplied title or if the work is known by one or several different names. Our sources for uniform titles are the Library of Congress Name Authority File (LCNAF), the Virtual International Authority File (VIAF), and the <i>Clavis Patrum Graecorum</i> and <i>Clavis Patrum Latinorum</i>. In some cases, we have established project-specific uniform titles. To date, we have supplied uniform titles for all biblical and liturgical texts. The work on uniform titles for theological, historical, and other texts is ongoing and these are applied only where feasible and where uniform titles are already established. The uniform titles applied during this phase of the project will lay the foundation for a comprehensive title index that we plan to implement as we undertake additional scholar-led description activities.
+        </p>
+      </div>
+
+      <div class='static-page__row'>
+        <h3 class='static-page__subtitle static-page__subtitle--sinai'>Contents Note</h3>
+        <p class='static-page__text static-page__text--sinai'>
+          The “Contents note” field allows us to provide more expansive descriptions of a manuscript's textual works. In many cases, especially where there are multiple works within a single bound manuscript object, Kamil’s entries provide more of a description than a title. In these cases, we use in the “Works” field what appears to be the primary work (or works) within Kamil’s entry. We then add Kamil’s longer description to the “Contents note” field. When primary works cannot be identified, or where there are too many to include in the “Works” field, we opt for the general title, Varia, following Kamil’s practice, leaving Kamil’s full description in “Contents note.” Similarly, Meimaris often provides both a general thema and a listing of contents for a manuscript. In some cases we are able to include each of the contents as titles in the “Works” field. In many others, however, we can only include the thema in “Works,” adding the fuller contents list to the “Contents note” field.
+        </p>
+
+        <p class='static-page__text static-page__text--sinai'>
+          When performing a “Title” search on the Sinai Manuscripts Digital Library website, both the “Works” field and the “Contents note” fields are searched.
```

Comment:
Sorry - I forgot to edit this bc we just introduced a new field. This line should be: When performing a "Title" search on the Sinai Manuscripts Digital Library website, the "Works', "Contents", and "Contents note" fields are searched.